topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md

```diff
@@ -211,62 +225,71 @@ As we directly see, the peak file lacks `chr` before any chromosome number. But
 
 > ### {% icon hands_on %} Hands-on: View end of file
 >
-> 1. **Select last** {% icon tool %}: Run **Select last lines from a dataset (tail)** with the following settings:
->     - **Text file** to our peak file `GSE37268_mof3.out.hpeak.txt`
->     - **Operation**: `Keep last lines`
->     - **Number of lines**: Choose a value, e.g. `100`
+> 1. Search for **Select last** {% icon tool %} tool and run **Select last lines from a dataset (tail)** with the following settings:
+>     - *"Text file"*: our peak file `GSE37268_mof3.out.hpeak.txt`
+>     - *"Operation"*: `Keep last lines`
+>     - *"Number of lines"*: Choose a value, e.g. `100`
 > 2. Click **Execute**
 > 3. Wait for the job to finish
-> 4. Inspect the file through the **eye icon**
+> 4. Inspect the file through the {% icon galaxy-eye %} (eye) icon (**View data**)
 >
 >    > ### {% icon question %} Questions
 >    >
->    > 1. Are the chromosomes 20 and 21 named X and Y?
+>    > 1. How are named the chromosomes?
+>    > 2. How are named the chromosomes X and Y?
```

Comment:
How are named the chromosomes? -> How are the chromosomes named?
How are named the chromosomes X and Y? -> How are the chromosomes X and Y named?